### ISSUE TITLE
Debug Email Process

### ISSUE THREAD

**Issue by taycom** [opencve/opencve#107]
## Expected Behavior
Is possible verify the configuracion or the process when email are send, in console, or change in debug mode for view the messages? 

## Actual Behavior
Configuration SMPT is Ok, but use relay ( no password, or user is required for send email ).

## Steps to Reproduce the Problem

  - 
  - 
  - 

## Specifications

  - OpenCVE version: Last
  - Python version: Python 3.7.3
  - Platform: Linux

**Comment by ncrocfer** [comment]
What kind of debug messages would you like to see ? Are you talking about the API part (reset password, receive confirmation mail for registration...) or in Celery (send reports) ?

**Comment by surfer190** [comment]
In my case the API part you mention works correctly but the celery part does not

Looking at the results backend in redis I see the tasks being successful:

"{\"status\": \"SUCCESS\", \"result\": null, \"traceback\": null, \"children\": [[[\"c93ef583-d855-4bea-9694-157bcab3e5ad\", null], null]], \"task_id\": \"c45e39dc-4005-41e8-aacb-7d24de529c15\", \"date_done\": \"2021-09-14T01:03:09.254988\", \"parent_id\": \"33996333-cf3a-4557-a9ca-81f5055db165\"}"

How can I dig a bit deeper to see where the problem lies? Can I log the task output?
@ncrocfer

**Comment by ncrocfer** [comment]
@surfer190 you say the Celery part does not work but you show me a typical result of a Celery task saying it succeeded...

> How can I dig a bit deeper to see where the problem lies?

What is your problem ? Please attach your worker logs.

**Comment by surfer190** [comment]
Yeah the task succeeded - but I was trying to figure out why the email alerts were not sending.
I think the problem was the smtp username and password set to `None` in the config instead of leaving empty was the cause.

Will have to check the worker logs to verify.

**Comment by ncrocfer** [comment]
Yes checking your logs will give you information about your errors:

https://github.com/opencve/opencve/blob/f7a9c4df50d039541c1aabd8f20e74070febbd07/opencve/tasks/reports.py#L175

**Comment by sbkg0002** [comment]
How can you verify if the settings are correct without waiting for a event to occur?

**Comment by ncrocfer** [comment]
You can use the forget password form.

**Comment by ncrocfer** [comment]
Closing this issue as it has been inactive for a long time (for new OpenCVE v2 users, use this doc: https://docs.opencve.io/guides/smtp_configuration/).
